package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "rm -rf dist/ && tsc --build --verbose",
+    "build": "rm -rf dist/ && tsc --build --verbose && ./build.sh",
```

Comment:
Comparing the JS repo, I think we can lose the `&& tsc --build --verbose` here since the `tsc` at the end of build.sh should cover us. Mind testing that?